stories/__tests__/SQFormMultiSelect.stories.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import * as Yup from 'yup';
+import {render, screen, waitFor, within} from '@testing-library/react';
+import {composeStories} from '@storybook/testing-react';
+
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormMultiSelect.stories';
+
+const {SQFormMultiSelect} = composeStories(stories);
+
+const initialDropdownValue = '- -';
+
+afterEach(() => {});
+
+describe('Tests for SQFormMultiSelect', () => {
+  it('should render a form with a multiselect input and dropdown options', async () => {
+    render(<SQFormMultiSelect size="auto" />);
+
+    const expandButton = await waitFor(() =>
+      screen.findByRole('button', {
+        name: /friends/i
+      })
+    );
+
+    userEvent.click(expandButton);
+
+    const optionsList = await screen.findByRole('listbox');
+    await waitFor(() => expect(optionsList).toBeInTheDocument());
+
+    const options = await within(optionsList).findAllByRole('option');
+    await waitFor(() => expect(options).toHaveLength(17)); // initial '- -' + 16 MOCK_FRIENDS_OPTIONS
+  });
+
+  it('should render the multiselect with default initial value "- -"', async () => {
+    render(<SQFormMultiSelect size="auto" />);
+
+    const expandButton = await waitFor(() =>
+      screen.findByRole('button', {name: /friends/i})
+    );
+    await waitFor(() =>
+      expect(expandButton).toHaveTextContent(initialDropdownValue)
+    );
+  });
+
+  it('should show empty value "- -" if provided initial value not in options', async () => {
+    render(<SQFormMultiSelect size="auto" initialValue={{friends: [0]}} />);
```

Comment:
`initialValues` isn't a prop on `SQFormMultiSelect`, but I think I see what you were going for. In some of the other tests you can pass in different `initialValues` to the underlying `SQForm` because the story was set up to make that possible with `SQFormProps`. This story does not have that set up because it existed before that solution was created.

Check out the stories and tests for `SQFormAutocomplete` to get an idea of how to set that up. Feel free to ping me if you have any questions - the storybook template and args structure is kind of weird.

After that, a test for checking what should happen when an initial value is provided that _is_ in the options would be good to add.